extensions/amp-story/0.1/amp-story.js

```diff
@@ -227,10 +228,6 @@ export class AmpStory extends AMP.BaseElement {
       enXaPseudoLocaleBundle
     );
 
-    registerServiceBuilder(this.win, 'localization-v01', function () {
-      return localizationService;
-    });
```

Comment:
What is "localization-v01" service? I don't see it referenced elsewhere in this PR.

Reply:
Was saving it for a future PR to delete these services only used in amp-story-0.1, which have been deprecated. But I just included those changes in this PR as well.